src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1093,29 +1097,28 @@ export function makeAxisDataPromise_Molecular_MakeMutationData(
     molecularProfileIds: string[],
     hugoGeneSymbol: string,
     mutations: Pick<
-        Mutation,
-        'uniqueSampleKey' | 'proteinChange' | 'mutationType'
+        AnnotatedMutation,
+        'uniqueSampleKey' | 'proteinChange' | 'mutationType' | 'putativeDriver'
     >[],
     coverageInformation: CoverageInformation,
     mutationCountBy: MutationCountBy,
     samples: Pick<Sample, 'uniqueSampleKey'>[]
 ) {
+    const sampleToMutations = _.groupBy(mutations, m => m.uniqueSampleKey);
     // collect mutations by sample by type
-    const sampleToMutationTypes = _.mapValues(
-        _.groupBy(mutations, m => m.uniqueSampleKey),
-        sampleMuts =>
-            _.uniq(
-                sampleMuts.map(
-                    m => mutationTypeToDisplayName[getOncoprintMutationType(m)]
-                )
+    const sampleToMutationTypes = _.mapValues(sampleToMutations, sampleMuts =>
+        _.uniq(
+            sampleMuts.map(
+                m => mutationTypeToDisplayName[getOncoprintMutationType(m)]
```

Comment:
should we generalize the function name here?   seems like the mutationtype is not just oncoprint related?

Reply:
I agree but for now I think it's okay as a shorthand for a certain taxonomy. we already have "simplified mutation type" which is something else. do you have another idea?